database/factories/RoleFactory.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Database\Factories;
+
+use App\Models\Role;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+class RoleFactory extends Factory {
+
+    /**
+     * The name of the factory's corresponding model.
+     *
+     * @var string
+     */
+    protected $model = Role::class;
+
+    public function definition()
+    {
+        return [
+          'name' => $this->faker->streetName(),
+        ];
+    }
+}
```

Comment:
Adcionar quebra de linha ao fim do arquivo:

```suggestion
}

```